.github/workflows/publish-hadoop-lakefs.yaml

```diff
@@ -0,0 +1,54 @@
+name: publish hadoop lakefs
+
+on:
+  release:
```

Comment:
I think this workflow should be triggered only using "workflow_dispatch".
Also the version number (I think) should not be coupled with the lakeFS version.

When the developer wants to publish a new version, they change the version in pom manually, and run this workflow.

WDYT?

Reply:
will remove